multicluster/cmd/service-mirror/main.go

```diff
@@ -204,6 +204,8 @@ func restartClusterWatcher(
 		repairPeriod,
 		probeWorker.Liveness,
 		enableHeadlessSvc,
+		// always instantiate the gatewayAlive=0 to prevent unexpected service fail fast
```

Comment:
nit: would it make more sense for `gatewayAlive=1` here? generally true evaluates to the constant 1, and false to 0.

Reply:
Sorry for the mistake, I'll correct this comment.

Reply:
No problem! :)